### PR TITLE
[Docs: Integration Test Readme] Updated with proper commands

### PR DESCRIPTION
**Pull request by sbis04**
I have updated the proper commands for initializing the `IntegrationTestWidgetsFlutterBinding` in the [integration test readme](https://github.com/flutter/flutter/tree/master/packages/integration_test#screenshots) screenshots section.

## Previously

```dart
void main() {
  final IntegrationTestWidgetsFlutterBinding binding =
      IntegrationTestWidgetsFlutterBinding.ensureInitialized();

  // ...
}
```

This produces the following linting error:

```
A value of type 'WidgetsBinding' can't be assigned to a variable of type 'IntegrationTestWidgetsFlutterBinding'.
Try changing the type of the variable, or casting the right-hand type to 'IntegrationTestWidgetsFlutterBinding'.
```

## Updated

```dart
void main() {
  final IntegrationTestWidgetsFlutterBinding binding =
      IntegrationTestWidgetsFlutterBinding();

  // ...
}
```

This was pointed out in the issue comment https://github.com/flutter/flutter/issues/86985#issuecomment-886571531

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by sbis04** [comment]
@HansMuller @goderbauer @Hixie Can anyone please take a look at this PR?

**Review by dnfield — CHANGES_REQUESTED** [review]
This is probably missing a cast, but I don't think removing the `ensureInitialized` is correct.

File: packages/integration_test/README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ This feature is currently supported on Android, iOS, and Web.
 ```dart
 void main() {
   final IntegrationTestWidgetsFlutterBinding binding =
-      IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+      IntegrationTestWidgetsFlutterBinding();
```
Comment (dnfield) [inline packages/integration_test/README.md:112]:
```suggestion
      IntegrationTestWidgetsFlutterBinding.ensureInitialized() as IntegrationTestWidgetsFlutterBinding;
```

File: packages/integration_test/README.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ Future<void> main() async {
...
```
Comment (dnfield) [inline packages/integration_test/README.md:157]:
```suggestion
      IntegrationTestWidgetsFlutterBinding.ensureInitialized() as IntegrationTestWidgetsFlutterBinding;
```

File: packages/integration_test/README.md (hunk 1 above)
Comment (johnpryan) [reply packages/integration_test/README.md:112]: Also this cast the type declaration can be removed:

```dart
final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized() as IntegrationTestWidgetsFlutterBinding;
```

File: packages/integration_test/README.md (hunk 2 above)
Comment (johnpryan) [reply packages/integration_test/README.md:157]: Same as above, the type declaration can be removed I think:

```dart
final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized() as IntegrationTestWidgetsFlutterBinding;
```

File: packages/integration_test/README.md (hunk 1 above)
Comment (sbis04) [reply packages/integration_test/README.md]: Have updated it

File: packages/integration_test/README.md (hunk 2 above)
Comment (sbis04) [reply packages/integration_test/README.md]: Have updated it

**Comment by sbis04** [comment]
@dnfield @johnpryan I have updated it to the following and also tested it on the latest `master`:

```dart
final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized() as IntegrationTestWidgetsFlutterBinding;
```

**Review by dnfield — APPROVED** [review]

**Review by johnpryan — APPROVED** [review]

**Pull request merged**